docs/kubernetes_azureStorage.md

```diff
@@ -0,0 +1,163 @@
+
```

Comment:
Can we change the title to use the underscore convention? `azureStorage` -> `azure_storage`.

Reply:
I added the file kubernetes_ayureStorage.md by mistake already in this pull request...

This is work in progress.

I will address all you suggestions in file kubernets_basic and then do a new PR